tests/test_filtsmooth/test_statespace/test_continuous/test_linearsdemodel.py

```diff
@@ -48,21 +48,12 @@ def test_diff(self):
         self.assertLess(np.linalg.norm(diffusion - self.diffmat), 1e-14)
 
     def test_ndim(self):
-        self.assertEqual(self.lm.ndim, TEST_NDIM)
-
-    def test_sample(self):
-        samp = self.lm.sample(0.0, 1.0, 0.1, (np.ones(TEST_NDIM)))
-        self.assertEqual(samp.ndim, 1)
-        self.assertEqual(samp.shape[0], TEST_NDIM)
-
-    def test_chapmankolmogorov(self):
-        """
-        Test if CK-solution for a single step
-        is according to iteration.
-        """
+        self.assertEqual(self.lm.dimension, TEST_NDIM)
+
+    def test_transition_rv(self):
         mean, cov = np.ones(TEST_NDIM), np.eye(TEST_NDIM)
         rvar = Normal(mean, cov)
-        cke, __ = self.lm.chapmankolmogorov(0.0, 1.0, 1.0, rvar)
+        cke, _ = self.lm.transition_rv(rv=rvar, start=0.0, stop=1.0, step=1.0)
```

Comment:
Ok weird question, but why actually pass `step`? Does anyone still use that, or do all functions below just compute `stop-start` anyways? This seems confusing to me, and I have the impression that we might want to just use `start` and `stop` everywhere and remove most mentions of `step`.

Reply:
`step` is relevant for the solution of those chapman-kolmogorov ODE things. In this test, however, solving them with a single step makes it easier to test. I think...

Reply:
Thanks for clearing that up, now I finally understand what `step` is actually about :D Ich somehow always interpreted it as the same as the "step size" in the ODE filters, but it makes sense of course to have a `step` argument if underneath there is another Euler method that requires a step size :)

Reply:
I see... I think in this case we should take care of this problem and make it more explicit. (Unresolving this convo, sorry haha). If I rename it to `euler_step`, or `aux_step` or sth, would that help?

Reply:
I resolved it because it felt like it "clicked" and I changed my point of view on these CKE things, but you're right that we should maybe make it more explicit to prevent future misunderstandings, so good call on that! I actually also thought about `euler_step` :D So between your proposals I think I'd prefer that, but if you think we should keep it more general than just for Euler methods then use `aux_step` or something else.

Reply:
I think `euler_step` is good. It is extremely precise and I don't want to guarantee that we might want to change things here at some point, but we catch it in those `**kwargs` anyway and here this might be a good use. I will rename to `euler_step`.